eng/performance/benchmark_jobs.yml

```diff
@@ -123,7 +123,7 @@ jobs:
               ${{ channel }}:
                 _Channel: ${{ channel }}
                 _Configs: CompilationMode=Tiered RunKind="${{ parameters.kind }}"
-                _PerfLabArguments: $(PerfLabArguments)
+                _PerfLabArguments: $(PerfLabArguments) # not necessary?
```

Comment:
It looks like we do still use this: https://github.com/dotnet/performance/blob/main/eng/performance/benchmark_jobs.yml#L99